docs/data/material/components/buttons/buttons.md

```diff
@@ -113,12 +113,28 @@ Use `color` prop to apply theme color palette to component.
 
 {{"demo": "IconButtonColors.js"}}
 
+### Loading with badge
```

Comment:
I think it would be better to separate loading and badge demos. Is there a reason to mix both?

Reply:
Done. The reason was to ensure a case like https://github.com/mui/material-ui/pull/43436, but it's better to separate.